--- hedgewars-src-1.0.3/project_files/hwc/rtl/sysutils.c~	2025-07-22 19:12:55.000000000 +0200
+++ hedgewars-src-1.0.3/project_files/hwc/rtl/sysutils.c	2026-06-07 10:18:51.630501943 +0200
@@ -83,7 +83,7 @@
 // Semi-dummy implementation of FormatDateTime
 string255 fpcrtl_formatDateTime(string255 FormatStr, TDateTime DateTime)
 {
-    string255 buffer = STRINIT(FormatStr.str);
+    string255 buffer = FormatStr;
     time_t rawtime;
     struct tm* my_tm;
 
