--- hedgewars-src-0.9.14.1/misc/hedgewars.desktop~	2010-11-14 19:17:01.000000000 +0100
+++ hedgewars-src-0.9.14.1/misc/hedgewars.desktop	2010-11-16 11:29:06.756836667 +0100
@@ -1,7 +1,5 @@
 [Desktop Entry]
 Type=Application
-Version=1.0
-Encoding=UTF-8
 Name=Hedgewars
 GenericName=Fighting Hedgehogs
 GenericName[de]=Kämpfende Igel
@@ -12,8 +10,9 @@
 GenericName[es]=Batallas entre erizos
 GenericName[it]=Ricci combattenti
 GenericName[pt]=Batalhas entre ouriços
-Icon=hedgewars.png
+Icon=hedgewars
 Exec=hedgewars
 Terminal=false
 StartupNotify=false
-Categories=Application;Game;StrategyGame;
+Categories=Game;StrategyGame;
+# vi: encoding=utf-8
